src/frontend/src/lib/services/auth.services.ts

```diff
@@ -104,10 +104,11 @@ const emptyIdbAddress = async (deleteIdbAddress: (principal: Principal) => Promi
 	}
 };
 
-const emptyIdbBtcAddressMainnet = async () => emptyIdbAddress(deleteIdbBtcAddressMainnet);
+const emptyIdbBtcAddressMainnet = (): Promise<void> => emptyIdbAddress(deleteIdbBtcAddressMainnet);
 
-const emptyIdbEthAddress = async () => emptyIdbAddress(deleteIdbEthAddress);
+const emptyIdbEthAddress = (): Promise<void> => emptyIdbAddress(deleteIdbEthAddress);
 
+// eslint-disable-next-line require-await
```

Comment:
same question here

Reply:
For simplicity reason. This function, `clearTestnetsOption`, execution is grouped with `emptyIdbBtcAddressMainnet` and `emptyIdbEthAddress`  in an existing `await Promise.all`. The two "idb" functions have to be promised.

Reply:
right! thank you!